simtools/simulator.py

```diff
@@ -224,7 +235,9 @@ def _load_sim_tel_config_and_model(self, config_data):
             "parameters", "array-simulator_parameters.yml"
         )
         _parameters = gen.collect_data_from_yaml_or_dict(_parameter_file, None)
-        self.config = gen.validate_config_data(_rest_config, _parameters)
+        # The solution below with ignore_unidentified might be temporary
+        # TODO - Think if there's a better solution
+        self.config = gen.validate_config_data(_rest_config, _parameters, ignore_unidentified=True)
```

Comment:
Don't see anything bad about this solution.

Reply:
Yeah I guess it's not too bad. Do you think I can remove the TODO then?

Reply:
Yes, remove the TODO.